Makefile

```diff
@@ -99,22 +99,23 @@ endif
 	rm -f test/copyimg/copyimg
 	rm -f test/checkseccomp/checkseccomp
 
-crioimage:
-	docker build -t ${CRIO_IMAGE} .
+libpodimage:
+	docker build -t ${LIBPOD_IMAGE} .
 
-dbuild: crioimage
-	docker run --name=${CRIO_INSTANCE} --privileged ${CRIO_IMAGE} -v ${PWD}:/go/src/${PROJECT} --rm make binaries
+dbuild: libpodimage
+	docker run --name=${LIBPOD_INSTANCE} --privileged ${LIBPOD_IMAGE} -v ${PWD}:/go/src/${PROJECT} --rm make binaries
 
-integration: crioimage
-	docker run -e STORAGE_OPTIONS="--storage-driver=vfs" -e TESTFLAGS -e TRAVIS -t --privileged --rm -v ${CURDIR}:/go/src/${PROJECT} ${CRIO_IMAGE} make localintegration
+integration: libpodimage
+	docker run -e STORAGE_OPTIONS="--storage-driver=vfs" -e TESTFLAGS -e TRAVIS -t --privileged --rm -v ${CURDIR}:/go/src/${PROJECT} ${LIBPOD_IMAGE} make localintegration
 
 testunit:
 	$(GO) test -tags "$(BUILDTAGS)" -cover $(PACKAGES)
 
-localintegration: clean binaries test-binaries
+localintegration: binaries test-binaries
 	./test/test_runner.sh ${TESTFLAGS}
```

Comment:
@mheon I remove ``clean`` from this.  That allowed tests to run (b/c clean removes bin/common).  But maybe that was bad to do.

Reply:
Binaries should be rebuilding conmon. It didn't look like it was. I'll look into it.

Reply:
I am removing both the clean and the binaries.  Currently we don't build anywhere but make all or make binaries. lets keep it that way.

Reply:
kk, so ``make -C $GOSRC binaries install.tools all TAGS=${TAGS}``  and I commented out the original one.